docker-compose.yml

```diff
@@ -13,6 +13,7 @@ services:
     command: >
       sh -c "python manage.py wait_for_db &&
              python manage.py migrate &&
+            python manage.py loaddata sample_cinema.json &&
```

Comment:
Please, make all commands aligned the same, they should have the same count of spaces before starting